{
  "id": "ch_CSY11CK7T_2024-12-04_1733344985.076219_thread",
  "type": "channel",
  "channel_name": "discuss-frontend",
  "conversation_type": "thread",
  "participants": [
    "Liam Callanan",
    "Serena"
  ],
  "messages": [
    {
      "sender": "Liam Callanan",
      "user_id": "U03BXSHAK0T",
      "message": "Also looks like most tests are failing right now because they’re hanging at `deleteProjectIfItExists`",
      "time": "12:43",
      "timestamp": "1733344985.076219",
      "is_reply": false
    },
    {
      "sender": "Serena",
      "user_id": "U01CZ3LBFU4",
      "message": "which run is this from?",
      "time": "14:11",
      "timestamp": "1733350306.166469",
      "is_reply": true
    },
    {
      "sender": "Serena",
      "user_id": "U01CZ3LBFU4",
      "message": "i assume you’re talking about https://cloud.cypress.io/projects/cfhedy/runs/16048/overview/b7038fb3-6089-4779-9f83-[…]ORED&runStatus=NOTESTS&runStatus=TIMEDOUT&ts=1733340290553.1 (https://cloud.cypress.io/projects/cfhedy/runs/16048/overview/b7038fb3-6089-4779-9f83-516dec0f4ca9/replay?att=1&roarHideRunsWithDiffGroupsAndTags=1&runStatus=PASSED&runStatus=FAILED&runStatus=RUNNING&runStatus=CANCELLED&runStatus=ERRORED&runStatus=NOTESTS&runStatus=TIMEDOUT&ts=1733340290553.1)?\n\ni see `Failed to get all BUILD projects for cypress-user-65786_instabase.com (http://cypress-user-65786_instabase.com): Error occurred executing GRPC method: Request /ai_hub_service.AIHubService/GetProjects failed. Error code=INTERNAL, Error=(13, 'internal'), Message: Could not read projects. Non-retriable error` as the error message returned by https://crafting-cyp--pr-65786-cypress-instabase.instabase.site.sandboxes.run/api/v2/aihub/build/projects?query_option=creator\n\nand i also see a lot of `Org not found` errors in the core-platform-service logs (https://crafting-cyp.test.instabase.com/grafana/explore?schemaVersion=1&panes=%7B%22u44%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bcontainer%3D%5C%22core-platform-service%5C%22%7D%20%7C%3D%20%60cypress-user-65786_instabase.com%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%22now-6h%22,%22to%22:%22now%22%7D%7D%7D&orgId=1)\n\nit seems that the user/org setup is off for this CI run?",
      "time": "14:52",
      "timestamp": "1733352761.746859",
      "is_reply": true
    },
    {
      "sender": "Serena",
      "user_id": "U01CZ3LBFU4",
      "message": "i triggered a Cypress CI rerun for this PR, and the `make-sandbox` step succeeded\n\nbut there are still a lot of `Org (cypress-user-65786_instabase.com (http://cypress-user-65786_instabase.com)) not found` and `Org (support-testing.1_instabase.com (http://1_instabase.com)) not found` errors in the test results :disappointed:\n\n@Xindi Xu how should we debug this?",
      "time": "15:12",
      "timestamp": "1733353962.822819",
      "is_reply": true
    },
    {
      "sender": "Serena",
      "user_id": "U01CZ3LBFU4",
      "message": "oh oops nvm i think these are caused by a bug in app runs, which is fixed by https://github.com/instabase/instabase/pull/65884\n\nno need for you to investigate then! @Xindi Xu",
      "time": "15:13",
      "timestamp": "1733354031.021389",
      "is_reply": true
    }
  ],
  "metadata": {
    "channel_id": "CSY11CK7T",
    "channel_name": "discuss-frontend",
    "date_file": "2024-12-04.json",
    "message_count": 5,
    "start_time": "1733344985.076219",
    "end_time": "1733354031.021389",
    "is_thread": true
  }
}